### PR TITLE
ENH: Enhanced several components in freedom-rains shop page

### PR DESCRIPTION
**Pull request by callmejim1226**
Improved several components, such as button design, filtering, image container (in the product page), and icon image, in the freedom-rains shop page. 

**Notes:** I wasn't able to manually see how toggling photos works on the shop product's page because when I got 264 products by getAllProducts request in postman, none of the products had more than one image's URL in its images field. Instead, I manually modified a code (for testing purposes) on the product page and took a screenshot of the product page to show how it will look when the product has more than one image.

**Screenshot of the product page when the product has more than one image:**
ticket_#3ygdxqj_SS1

### CODE REVIEW

**Comment by abstudios617** [comment]
Task linked: [CU-3ygdxqj Shop](https://app.clickup.com/t/3ygdxqj)

File: client/src/components/product-filter/index.js [hunk 1]
```diff
@@ -273,6 +281,24 @@ const ProductFilter = ({
...
+  };
+
+  const handleStoreClick = () => {
+    storeList ? setStoreList(false) : setStoreList(true); 
+  };
+
+  const handlePriceClick = () => {
+    priceList ? setPriceList(false) : setPriceList(true); 
+  };
+
```
Comment (stephull) [inline client/src/components/product-filter/index.js:299]: For the handler functions, we could proably change it to, for instance,:
`const handlePlatformClick = () => {
    setPlatformList((state) => !state);
}`
state being the current true/false value

File: client/src/components/product-filter/index.js [hunk 2]
```diff
@@ -473,7 +505,7 @@ const ProductFilter = ({
         <FilterContainer>
           <FilterTitle>
             Price Range
-            <img src={vector} alt="vector" />
+            <img src={vector} alt="vector" onClick={() => handlePriceClick()} />
           </FilterTitle>
```
Comment (stephull) [inline client/src/components/product-filter/index.js:509]: Each FilterTitle's onClick can be shortened to `onClick={handlePriceClick}`

**Review by stephull — APPROVED** [review]
I commented some recommendations for **/product-filter/index.js**, but otherwise everything looks fine 👍

**Review by AlamTahmidul — APPROVED** [review]

**Pull request merged**